src/wheel/macosx_libfile.py

```diff
@@ -352,13 +352,16 @@ def calculate_macosx_platform_tag(archive_root, platform_tag):
     base_version = tuple([int(x) for x in base_version.split(".")])
     if len(base_version) >= 2:
         base_version = base_version[0:2]
```

Comment:
```suggestion
    base_version = base_version[0:2]
```

There is no need to check the length.